test/backend/QueryScheduler.spec.js

```diff
@@ -550,7 +550,6 @@ describe('QueryScheduler', function() {
             queryObject = {
                 fid,
                 uids,
-                refreshInterval,
```

Comment:
why?

Reply:
I believe Brian just wanted to test that a default refresh interval was set even if one isn't sent with the request. 

I can add it back in and make the assertion:
```js
assert.deepEqual(
    getQueries(),
    [queryObject],
    'Query has not been saved'
);
```
would you prefer that?

Reply:
Yes, please. Otherwise I won't know if this PR changes Falcon's behaviour.